src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -1272,7 +1272,7 @@ const formConfig = {
               get(
                 'applicantMedicareStatus.eligibility',
                 formData?.applicants?.[index],
-              ) !== 'enrolled' &&
+              ) === 'ineligible' &&
```

Comment:
There are three possible options:
- `enrolled`
- `ineligible` <- this is the only one we should check against
- `eligibleNotSignedUp` <- this was the one that caused the false positive